src/psyclone/dynamo0p3.py

```diff
@@ -776,10 +768,10 @@ def function_spaces(self):
         spaces, we return both. '''
         if self._type == "gh_field":
             return [self.function_space]
-        elif self._type in VALID_OPERATOR_NAMES:
+        elif self._type in GH_VALID_OPERATOR_NAMES:
```

Comment:
Again, this can just be `if` and same again below.

Reply:
Done.